docs/source/IC.rst

```diff
@@ -0,0 +1,22 @@
+IC
+=====
+
+IC stands for Invisible Cities
+*Language*: Python; *Based*: Geant4 toolkit, divided in logical blocks
```

Comment:
It's a repetition of the explanation of NEXUS.

Reply:
Fixed!